api/db/seeds/data/organizations-builder.js

```diff
@@ -120,4 +120,39 @@ module.exports = function organizationsBuilder({ databaseBuilder }) {
     birthdate: '2003-09-30',
     organizationId: 3,
   });
+
+  const userConnectedByEmailOnly = databaseBuilder.factory.buildUser.withUnencryptedPassword({
+    firstName: 'connect',
+    lastName: 'email',
+    email: 'connect.email@example.net',
+    username: null,
+    rawPassword: 'Pix123',
+    cgu: false
+  });
+  databaseBuilder.factory.buildSchoolingRegistration({
+    userId: userConnectedByEmailOnly.id,
+    firstName: userConnectedByEmailOnly.firstName,
+    lastName: userConnectedByEmailOnly.lastName,
+    birthdate: '2010-09-30',
+    samlId: userConnectedByEmailOnly.samlId,
+    organizationId: 3,
+  });
+
+  const userConnectedFromGarOnly = databaseBuilder.factory.buildUser.withUnencryptedPassword({
```

Comment:
Est-ce que les seeds seront laissés ? 
```suggestion
  const userConnectedWithSmalId = databaseBuilder.factory.buildUser.withUnencryptedPassword({
```

Reply:
Yes, on les gardes. Les noms prénoms ont été modifiés.